src/status_im/ui/screens/accounts/login/views.cljs

```diff
@@ -82,10 +82,10 @@
         (when-not can-navigate-back?
           [components.common/bottom-button
            {:label    (i18n/label :t/other-accounts)
-            :on-press #(re-frame/dispatch [:navigate-to-clean :accounts])}])
+            :on-press #(re-frame/dispatch [:navigate-to :accounts])}])
```

Comment:
this was a right behavior, it should be not possible to go back in this case, let's just revert this file

Reply:
@flexsurfer done.